airbyte-integrations/connectors/source-file-secure/setup.py

```diff
@@ -2,9 +2,20 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
 
+import os
+from pathlib import Path
 
 from setuptools import find_packages, setup
 
+
+def local_dependency(name: str) -> str:
```

Comment:
Compute the absolute path to the local dependencies. In dagger pipelines we mount local dependencies to `/local_dependencies`.

Reply:
Oh dear python!

I like this solution.

My only reservations are having
1. Dagger relying on `file://` to mark relative dependencies, or pip -e as this isnt always the case ([example](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/metadata_service/orchestrator/pyproject.toml#L18
))
2. our connector setup.py's relying on knowing how dagger works under the hood.

So I want to ask a question for #1 and propose a possible idea for 2

1. I dont think we can do away with this, but how come we dont mount the relative import to the same relative location in the dagger file tree?

2. What if, in the dagger machine, we symlinked the local host path to the dagger `/local_dependencies` version? Would remove the need for having `DAGGER_BUILD` in our connectors folder.